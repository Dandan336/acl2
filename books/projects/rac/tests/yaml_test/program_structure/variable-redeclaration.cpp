#include "ac_int.h"

// RAC begin

typedef ac_int<2, true> int2;

int foo()
{
    int a = 2;
    int2 a = 3;
    return a;
}

// RAC end
