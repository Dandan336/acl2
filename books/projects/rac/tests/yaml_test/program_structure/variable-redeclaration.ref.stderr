variable-redeclaration.cpp:10: Duplicate identifier declaration (could be an enum, a variable or template parameter) `a`
